src/Common/getQueryId.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include <Common/CurrentThread.h>
+
+namespace DB
+{
+
+static String getQueryId()
```

Comment:
Why can't you use `CurrentThread::getQueryId` directly, i.e. what's the usecase for this function?

Reply:
Just needed to make sure that query context is not nullptr as it is very important for my case. Indeed it should be guaranteed that there is query id <==> query context != nullptr, but seemed safer to check. But I think I will make it simply `getQueryId` as you suggest.